ironfish/src/storage/database/encoding.ts

```diff
@@ -172,6 +172,19 @@ export class BigIntLEEncoding implements IDatabaseEncoding<BigInt> {
   }
 }
 
+export class BigIntBEEncoding implements IDatabaseEncoding<BigInt> {
```

Comment:
this specifically is U64, we should put that in the encoding

Reply:
I'll rename this to `BigU64BEEncoding`